client/src/routes/fight/FighterBar.svelte

```diff
@@ -9,36 +9,32 @@
 </script>
 
 <div class="container">
-    <div class="center-icon">
-        <Icon icon="akar-icons:double-sword" width="4em" color="white" height="4em" />
-    </div>
-    
```

Comment:
Nie rozumiem powodu usunięcia tej ikony.

Reply:
Przepraszam, o tym musiałem napisać w głównych uwagach.

Ta ikonka przy zmianach parametru ekranu nie wyświetlała się w centrum górnego bloku. Próbowałem dostosować parametry CSS dla tej ikonki tak, żeby to się udało, ale cały czas coś mi nie grało, więc postanowiłem usunąć na czas prezentacji z klientem.

Pewnie warto byłoby po prostu skomentować, ale nawet nie pomyślałem.